server/src/service/PkiService.js

```diff
@@ -154,6 +154,50 @@ exports.getCurrentCARootCert = async function (envId) {
   };
 };
 
+/**
+ * Returns the Environment Status based on configured information for given envId
+ * envId String ID of environment
+ * returns inline_response_200_1
+ **/
+exports.getEnvStatus = async function (envId) {
```

Comment:
this logic is missing...it's hardcoded
and besides, shouldn't be in this file, which is related to PKI

Reply:
@chinogarcia Please review after adding logic per: https://github.com/modusbox/connection-manager-api/pull/24/commits/734410249d83e5e3a176b5273925b1b06e9649ef Please let me know where to place code if not at Pki? None of the existing make sense, let me know if you want me to create a new service module. Thank you.

Reply:
in that case, then yes, a new service should be created

Reply:
Now that it takes the dfspId, and therefore is the Environmnet's DFSP Status...moved it to DFSPNetworkConfig as it is status of its configuration.